src/lib/shell/MainLoopSiWx917.cpp

```diff
@@ -0,0 +1,208 @@
+/*
+ *

```

Comment:
Is there a significant change between MainLoopEfr32 and MainLoopSiWx917 to have this duplicate file? At first glance they look identical. And you are using the same streamer_ implementation so I would assume this shell task to be the same.

Reply:
The only change we are calling the streamer read and then waiting for the shell activity. Since that buffer is filled before we trigger that 

```
        if (streamer_read(streamer_get(), buffer + line_sz, 1) != 1)
        {
            continue;
        }
        chip::WaitForShellActivity();
```
and we are notifying the shell process, once the buffer is filled
```
    case ARM_USART_EVENT_RECEIVE_COMPLETE:
#ifdef ENABLE_CHIP_SHELL
        chip::NotifyShellProcessFromISR();
#endif;
```